### PR TITLE
Update dYdX config

### PR DESCRIPTION
**Pull request by lucadonnoh**

### CODE REVIEW

File: packages/config/src/layer2s/dydx.ts [hunk 1]
```diff
@@ -225,6 +225,28 @@ export const dydx: Layer2 = {
       description:
         'Allowed to update state of the rollup. When Operator is down the state cannot be updated.',
     },
+    {
+      name: 'Short Timelock Executor',
+      accounts: [discovery.getPermissionedAccount('Treasury', 'owner')],
```
Comment (michalsidzej) [inline packages/config/src/layer2s/dydx.ts:230]: What if this one changes? The owner field will be pointing to a different contract

Comment (michalsidzej) [reply packages/config/src/layer2s/dydx.ts:230]: Also, I don't think this is a permissioned account. This is a contract, no?

**Review by adamiak — APPROVED** [review]

**Review by lucadonnoh — COMMENTED** [review]
upgradability info in contracts is still missing, right?

**Review by lucadonnoh — COMMENTED** [review]
apart from the comments it lgtm

**Pull request merged**